diff --git a/tomviz/pybind11/CMakeLists.txt b/tomviz/pybind11/CMakeLists.txt
index 1234567..abcdefg 100644
--- a/tomviz/pybind11/CMakeLists.txt
+++ b/tomviz/pybind11/CMakeLists.txt
@@ -1,4 +1,4 @@
-find_package(TBB)
+find_package(TBB REQUIRED)

 set(CMAKE_MODULE_LINKER_FLAGS "")
 pybind11_add_module(_wrapping
@@ -22,10 +22,8 @@ pybind11_add_module(ctvlib
   ctvlib/eigenConversion.h
   ctvlib/WrappingCtvlib.cxx)
 target_link_libraries(ctvlib
-  PRIVATE tomvizcore VTK::eigen ${TBB_LIBRARIES})
+  PRIVATE tomvizcore VTK::eigen TBB::tbb)

 add_dependencies(ctvlib VTK::eigen)
-
-include_directories(ctvlib ${TBB_INCLUDE_DIR})

 set_target_properties(ctvlib PROPERTIES
